tests/unit/src/main/scala/tests/TestingServer.scala

```diff
@@ -785,6 +785,7 @@ final case class TestingServer(
     val abspath = toPath(filename)
     val oldText = abspath.toInputFromBuffers(buffers).text
     val newText = fn(oldText)
+    buffers.put(abspath, newText)
```

Comment:
why?

Reply:
We weren't actually updating the buffers